tests/ert/unit_tests/config/test_num_cpu.py

```diff
@@ -22,39 +20,50 @@ def test_num_cpu_from_config_preferred():
  {data_file_num_cpu} DISTRIBUTED/
 """
         )
-    config_dict = {
-        ConfigKeys.NUM_REALIZATIONS: 1,
-        ConfigKeys.NUM_CPU: config_num_cpu,
-        ConfigKeys.DATA_FILE: os.path.join(os.getcwd(), data_file),
-        ConfigKeys.ENSPATH: ".",
-        ConfigKeys.RUNPATH_FILE: os.path.join(os.getcwd(), "runpath.file"),
-    }
-    ert_config = ErtConfig.from_dict(config_dict)
-    assert ert_config.preferred_num_cpu == config_num_cpu
+    assert (
+        ErtConfig.from_dict(
+            {
+                ConfigKeys.NUM_CPU: config_num_cpu,
+                ConfigKeys.NUM_REALIZATIONS: 1,
+                ConfigKeys.DATA_FILE: data_file,
+            }
+        ).preferred_num_cpu
+        == config_num_cpu
+    )
 
 
 @pytest.mark.parametrize(
     "parallelsuffix", [("/"), (" /"), (" DISTRIBUTED/"), (" DISTRIBUTED /")]
 )
+@pytest.mark.parametrize(
+    "casetitle", ["CASE", "-- A CASE --", "PARALLEL Tutorial Case"]
```

Comment:
Including PARALLEL here is a nice touch! 👍🏻

Reply:
Out code probably allows an empty line also, should we include it in the tests but comment that ECLIPSE itself actually crashes on it?